### PR TITLE
Add global for SMTPSecure

### PR DESCRIPTION
**Pull request by mbutlerdti**
Add a global that maps to the SMTPSecure variable.

Let's you use SSL/TLS on SMTP connections.

Specifically had several clients using gmail, who requires secure connections.

### CODE REVIEW

**Comment by tmccormi** [comment]
This looks fine to me, if no one objects I'll merge it in this evening

File: library/globals.inc.php [hunk 1]
```diff
@@ -1155,6 +1155,17 @@
       xl('Must be empty if SMTP authentication is not used.')
     ),
 
+    'SMTP_SECURE' => array(
+      xlt('SMTP security protocol to connect with'),
+      array(
+        '' => 'None',
```
Comment (bradymiller) [inline library/globals.inc.php]: translate:
``````
'' => xl('None'),

File: library/globals.inc.php [hunk 2]
```diff
@@ -1155,6 +1155,17 @@
       xl('Must be empty if SMTP authentication is not used.')
     ),
 
+    'SMTP_SECURE' => array(
+      xlt('SMTP security protocol to connect with'),
```
Comment (bradymiller) [inline library/globals.inc.php]: use xl()

File: library/globals.inc.php [hunk 3]
```diff
@@ -1155,6 +1155,17 @@
...
+      array(
+        '' => 'None',
+        'ssl'  => 'SSL',
+        'tls'  => 'TLS'
+      ),
+      '',
+      xla('SMTP security protocol to connect with. Required by some servers such as gmail.')
```
Comment (bradymiller) [inline library/globals.inc.php]: use xl()

**Comment by bradymiller** [comment]
Placed some comments to fix for translation.

**Comment by openemr** [comment]
This code was revised by @arnabnaha and committed to the codebase:
https://github.com/openemr/openemr/commit/2421a6ef52086e20b6285d8497e79460e656253f

**Pull request closed without merging**